src/app/cli/src/coda_inputs.ml

```diff
@@ -407,44 +326,12 @@ module Pending_coinbase = struct
   module V1 = struct
     include Coda_base.Pending_coinbase.Stable.V1
 
-    [%%define_locally
-    Coda_base.Pending_coinbase.
-      ( hash_extra
-      , oldest_stack
-      , latest_stack
-      , create
-      , remove_coinbase_stack
-      , update_coinbase_stack
-      , merkle_root )]
-
-    module Stack = Coda_base.Pending_coinbase.Stack
-    module Coinbase_data = Coda_base.Pending_coinbase.Coinbase_data
-    module Hash = Coda_base.Pending_coinbase.Hash
+    include (
+      Coda_base.Pending_coinbase :
+        module type of Coda_base.Pending_coinbase with type t := t )
```

Comment:
I prefer the `include` pattern over `define_locally` because it reduces the number of things you need to touch in the code whenever you add new items (or remove old ones) from an interface. One of the reasons the current signature/module/functor pattern is so annoying is that whenever you want to make an update to a module (in this example, add or remove a type from its interface), you need to find the `n` number of places where that signature is `with type ... := ...` and update those as well. In a similar fashion, `define_locally` has the same issue. Ultimately, at the end of the day, there is no semantic difference here since there is still a signature constraining which items can be seen from inside of this structure that's being built. As such, I would argue that `define_locally` should only be used when there is a signature being inferred from a module by the compiler.

If this is not a convincing argument, I am still willing to switch it back.

Reply:
I think having `define_locally` is still better. Can we stick with that?

Reply:
I don't have a strong preference either way. I think the advantage of `define_locally` is that it tells you concisely what items are being used, which isn't readily apparent from the module signature being `include`d.

Reply:
Ok, I will switch back to `define_locally`.